packages/ui/src/components/Node/table/UseObserverTable.tsx

```diff
@@ -88,15 +85,14 @@ export function useObserverTable({ id, setIsDataFetched, setItems, items, parent
       onReceive: (linkIds) => {
         // If we have enough initial items, attempt unsubscribe
         if (itemsRef.current.length >= initialScreenCount) {
-          subscription?.unsubscribe();
```

Comment:
Assuming this was due to "fix: resolve issue where the table Node fails to load data except on the first run" Howcome it did not work?

Reply:
1. The first time we run `Run`, everything works fine.
2. However, when the condition itemsRef.current.length >= initialScreenCount is met, unsubscribe() is called.
3. This means the subscription is completely canceled.
4. As a result, even if we run `Run` again later, the onReceive callback will not be triggered since the previous subscription has been canceled.
5. This leads to `loadMore.current()` being unable to fetch new data subsequently. 
React expects components to re-render multiple times, so the correct approach is to let React's useEffect cleanup mechanism handle the subscription cleanup.

Reply:
Ok! Good call removing it. It was not a remarkable performance win anyway 👍